js/layout-javascript/small-card-code.js

```diff
@@ -2256,6 +2256,12 @@ DynamicList.prototype.expandElement = function(elementToExpand, id) {
     var directoryDetailImageWrapper = elementToExpand.find('.small-card-list-detail-image-wrapper');
     var directoryDetailImage = elementToExpand.find('.small-card-list-detail-image');
 
+    // Get the size of the html offset
+    // This get into account phones with notch
+    var computedStyles = window.getComputedStyle(document.getElementsByTagName("html")[0]);
+    var htmlMarginTop = computedStyles.getPropertyValue('margin-top');
+    var toTop = parseInt(htmlMarginTop.replace('px',''), 10);
```

Comment:
The `.replace()` shouldn't be necessary.

```suggestion
    var toTop = parseInt(htmlMarginTop, 10);
```

![image](https://user-images.githubusercontent.com/290733/54710713-32931180-4b40-11e9-9c6e-c9d1debd061f.png)